packages/yoroi-extension/app/api/ada/lib/state-fetch/remoteFetcher.js

```diff
@@ -407,4 +410,21 @@ export class RemoteFetcher implements IFetcher {
       return res;
     }, {});
   }
+
+  getCatalystRoundInfo: CatalystRoundInfoRequest => Promise<CatalystRoundInfoResponse> = async (body) => {
+    const { BackendService } = body.network.Backend;
+    if (BackendService == null) throw new Error(`${nameof(this.getCatalystRoundInfo)} missing backend url`);
+    const isMainnet = /Mainnet/g.test(body.network.NetworkName)
+    const prefix = isMainnet ? '' : 'api/'
+    return await axios(
+      `${BackendService}/${prefix}v0/catalyst/fundInfo`,
+      {
+        method: 'get',
```

Comment:
the prefix `api/` only exist on `testnet` but when I remove the prefix on mainnet the response is 404 this mean that the route isn't exist 
on testnet `https://testnet-backend.yoroiwallet.com/api/v0/catalyst/fundInfo` => Working
on mainnet `https://iohk-mainnet.yoroiwallet.com/v0/catalyst/fundInfo/` => not working

Reply:
`curl https://iohk-mainnet.yoroiwallet.com/api/v0/catalyst/fundInfo/` works.

Reply:
@AhmedIbrahim336 , it's because the proxy of `BackendService` itself includes the `/api` part for mainnet. So both URLs contain `/api`, but in the code it has to be adapted for the values. We might fix it later, bit not a priority right now